semantics.json

```diff
@@ -81,6 +76,39 @@
       }
     ]
   },
+  {
+    "name": "behaviouralSettings",
```

Comment:
Is there a reason to not use behavioral here? Wasn't sure if intentionally following british english or just by accident